pkg/cvo/upgradeable.go

```diff
@@ -29,17 +29,18 @@ import (
 const (
 	adminAckGateFmt             string = "^ack-[4-5][.]([0-9]{1,})-[^-]"
 	upgradeableAdminAckRequired        = configv1.ClusterStatusConditionType("UpgradeableAdminAckRequired")
+	checkThrottlePeriod                = time.Minute
```

Comment:
What is the downside of reducing the checkThrottlePeriod further?

Reply:
Good question, I'm not entirely sure. Abhinav's message from https://github.com/openshift/cluster-version-operator/commit/04528144feb7a8141801bce591fda43d65acc48a that introduced the throttling does not help too much about why exactly `minUpdateCheckInterval` was chosen:

> Adds another long running sync like the availableUpdates that updates the operator with upgradeable conditions every minUpdateChekInterval.

`minUpdateCheckInterval` itself happened in this beast https://github.com/openshift/cluster-version-operator/pull/45 and there's too much content there to be useful, the commit message says just:

> * Avoid reconciling too often (exit before applying) when no spec change recorded

I'm not sure about how expensive the upgradeability checks are - that's one of the two reasons I can imagine throttling is useful (the other is suppressing user-observable flapping in busy periods).

Reply:
But the refactoring on this PR should make experimenting with this easier ;)